module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,27 @@
+
+#include "arithmeticAverage.hpp"
+#include <functional>
+#include <numeric>
+#include <vector>
+#include <math.h>
+
+double ArithmeticAverage(const std::vector<int> &first_,
+                         const std::vector<int> &second_) {
+  return (double)std::inner_product(std::begin(first_), std::end(first_),
```

Comment:
It doesn't change anything. You should just get std::vector<double> to implicit cast int to double in vector